lib/hammer_cli/settings.rb

```diff
@@ -20,7 +20,7 @@ def self.load_from_paths(files)
           load_from_file(File.join(full_path, 'cli_config.yml'))
           load_from_file(File.join(full_path, 'defaults.yml'))
           # load config for modules
-          Dir.glob(File.join(full_path, 'cli.modules.d/*.yml')).sort.each do |f|
+          Dir.glob(File.join(full_path, '**/*.yml')).sort.each do |f|
```

Comment:
This may potentially break plugins that have more yaml files than just the standard one. Not sure how it would work for example with `hammer-cli-import` https://github.com/Katello/hammer-cli-import/tree/master/config